Kainos-map/SQL scripts/initialise_db.sql

```diff
@@ -0,0 +1,21 @@
+DROP DATABASE IF EXISTS courses;
+CREATE DATABASE IF NOT EXISTS courses;
+USE courses;
+
+DROP TABLE IF EXISTS Course;
```

Comment:
If the database has just been dropped, do we need this?

Reply:
fair enoof

Reply:
🏖